plugins/package-managers/node/src/main/kotlin/Yarn2.kt

```diff
@@ -87,6 +88,9 @@ private enum class YarnDependencyType(val type: String) {
  * - *disableRegistryCertificateVerification*: If true, the `yarn npm info` commands called by this package manager will
  *   not verify the server certificate of the HTTPS connection to the NPM registry. This allows to replace the latter by
  *   a local one, e.g. for intercepting the requests or replaying them.
+ * - *corepackEnabled*: If true, this implementation assumes that Yarn has been installed via
```

Comment:
IMO the name of the property (and also partly the description) is a bit misleading now, as it sounds as if Corepack would only be supported at all if `corepackEnabled` is set to `true`.

I'd rather propose to rename this e.g. to `corepackOverride` or `forceCorepack` and explain that Corepack support is usually auto-detected, but can be overridden / enforced using this property.

Reply:
Renamed the property and reworded the description.